webui/react/src/pages/Default.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Navigate } from 'react-router-dom';
+
+import useFeature from 'hooks/useFeature';
+import { dashboardDefaultRoute, defaultRoute, rbacDefaultRoute } from 'routes';
+
+const Default: React.FC = () => {
+  const dashboardEnabled = useFeature().isOn('dashboard');
+  const rbacEnabled = useFeature().isOn('rbac');
+
+  if (rbacEnabled) {
+    return <Navigate to={rbacDefaultRoute.path} />;
+  } else if (dashboardEnabled) {
```

Comment:
Dashboard should take priority here, we should be able to delete the rbac default route once we launch dashboard.